odin/chain.json

```diff
@@ -279,8 +303,8 @@
   },
   "explorers": [
     {
-      "kind": "odin web",
-      "url": "https://mainnet.odinprotocol.io/",
+      "kind": "Runa",
+      "url": "https://runa.odinprotocol.io/",
       "tx_page": "https://mainnet.odinprotocol.io/transactions/${txHash}"
```

Comment:
should the tx_page URL be updated to runa at the same time?

Reply:
Agreed, yes. Will make a fix